react-native/screens/PostingCreationScreen.js

```diff
@@ -203,40 +217,63 @@ const PostingCreationScreen = props => {
               ref={descriptionInputRef}
             />
           </View>
-          <View style={ (windowHeight > 650) ? styles.switchRowBig : styles.switchRow}>
-            <View style={styles.switchColumn}>
-              <Text style={styles.switchTitle}>Request?</Text>
-              <Switch
-                style={styles.switch}
-                onValueChange={toggleRequestSwitch}
-                value={isRequestSwitchEnabled}
-                trackColor={{ false: "#767577", true: Colors.primary }}
-                thumbColor={isRequestSwitchEnabled ? Colors.primary : "#f4f3f4"}
+          <View style={styles.pickerRow}>
+            <View style={styles.pickerView}>
+              <RNPickerSelect
+                    placeholder={{}}
+                    items={[
+                      {label: 'Request', value: 'r'},
+                      {label: 'Offer', value: 'o'},
+                    ]}
+                    onValueChange={
+                      value =>
+                      {if (value == null) {
+                        setIsRequestSelected(previousState => previousState);
+                      }
+                      else {
+                        changeType(value);
+                      }
+                    }}
+                  />
+            </View>
+            <View style = {styles.pickerView}>
+              <RNPickerSelect
+                    placeholder={{}}
+                    items={[
+                      {label: 'Goods', value: 'g'},
+                      {label: 'Services', value: 's'},
+                    ]}
+                    onValueChange={
+                      value =>
+                      {if (value == null) {
+                        setIsGoodSelected(previousState => previousState);
+                      }
+                      else {
+                        changeCategory(value);
+                      }
+                    }}
               />
             </View>
-            <View style={styles.switchColumn}>
-              <Text style={styles.switchTitle}>Count</Text>
+          </View>
+          <View style={styles.countView}>
+            <View style={styles.countInputTitle}>
+                <Text style={styles.countInputTitleText}>
+                  Quantity:
+                </Text>
+              </View>
+            <View>
               <View style={styles.countInputView}>
                 <TextInput
                   style={styles.countInputText}
+                  placeholderTextColor={Colors.placeholder_text}
                   keyboardType='numeric'
                   returnKeyType='done'
-                  value={itemCount.toString()}
+                  value = '1'
```

Comment:
This makes the quantity always reset to '1' .  I think it should be the `value={itemCount.toString()}`

Reply:
Nice catch.  That change fixes it.